docs/development.md

```diff
@@ -2,136 +2,103 @@
 
 This guide is only relevant for those wanting to dig deeper or make changes to
 Faasm itself. If you'd just like to write and run functions, you can refer to
-the [set-up](setup.md) instructions. 
+the [set-up instructions](setup.md). 
 
-## Recommended Set-up
+## Recommended Set-up - Docker CLI
 
-The Faasm codebase is all standard C/C++ and Python, so should work on a range 
-of set-ups, however, the recommended configuration is:
-
-- Ubuntu 20.04
-- Python 3.8
-- Clang 10
-- CMake 3.13+
-- Ninja (rather than Make)
-
-## Code style
-
-Code style is checked as part of the CI build and uses the following
-configuration:
-
-- **C/C++** - [`clang-format`](https://clang.llvm.org/docs/ClangFormat.html)
-  with the [Mozilla C++
-  style](https://firefox-source-docs.mozilla.org/code-quality/coding-style/coding_style_cpp.html)
-- **Python** - [Black](https://github.com/psf/black) with an 80 char line limit
-- **Markdown** - wrap to 80 characters where possible (not enforced)
-- **CMake** - TBD
-- **Bash** - TBD
-
-## Checking out the repo
-
-For now, many locations rely on the code being located at
-`/usr/local/code/faasm` (note the lowercase "f"), with the latter two 
-directories _owned_ by the current user. This is annoying and something we will 
-fix in future. 
-
-You can either set this directory up directly, or just symlink it.
-
-Assuming you've checked out this code somewhere, you'll need to make sure
-submodules are up to date:
+Most development can be done using the containerised CLI. To get everything set
```

Comment:
I personally would have thought this was obvious, and is probably overkill given that these docs are aimed at people wanting to do development.

A more obvious place to mention this would be in the main `README.md`, where the quick-start assumes people can run docker-compose. Again, I would have thought this was obvious, but if you want to include the links then perhaps put them on there.